### PR TITLE
Support for notifications for efr32

### PR DESCRIPTION
**Pull request by mkardous-silabs**
#### Problem
* Support for the android chip tool

#### Change overview
* Adds support for notifications for the bluetooth connection

#### Testing
* Tested by repeatedly doing the commissioning with the android tool and the python controller

#### Fixed issues
* #7078
* #7571
* #8502

### CODE REVIEW

File: src/platform/EFR32/BLEManagerImpl.cpp [hunk 1]
```diff
@@ -502,23 +508,31 @@ bool BLEManagerImpl::SendIndication(BLE_CONNECTION_OBJECT conId, const ChipBleUU
...
 
     VerifyOrExit(((conState != NULL) && (conState->subscribed != 0)), err = CHIP_ERROR_INVALID_ARGUMENT);
     VerifyOrExit(timerHandle != kMaxConnections, err = CHIP_ERROR_NO_MEMORY);
 
-    // start timer for light indication confirmation. Long delay for spake2 indication
-    sl_bt_system_set_soft_timer(TIMER_S_2_TIMERTICK(6), timerHandle, true);
-
-    ret = sl_bt_gatt_server_send_indication(conId, cId, (data->DataLength()), data->Start());
+    // start timer for light notification confirmation. Long delay for spake2 indication
+    sl_bt_system_set_soft_timer(TIMER_S_2_TIMERTICK(10), timerHandle, true);
```
Comment (jmartinez-silabs) [inline src/platform/EFR32/BLEManagerImpl.cpp:517]: Based on our converstions , is the longer delay from 6 to 10 still needed?

**Review by jmartinez-silabs — APPROVED** [review]

File: src/platform/EFR32/BLEManagerImpl.cpp (hunk 1 above)
Comment (mkardous-silabs) [reply src/platform/EFR32/BLEManagerImpl.cpp]: No, it isn't. Changed it back to 6 ms.

**Review by andy31415 — APPROVED** [review]

**Comment by andy31415** [comment]
@jmartinez-silabs - since this is device-specific code, would you be able to do the technical review?

**Comment by cjszz2001** [comment]
What is your test environment?
What version OTBR and RCP ?

**Comment by mkardous-silabs** [comment]
@cjszz2001 
OTBR commit hash: 15d556e94b72144db117b7d602418fe8f09a642f
For the rcp version, it's the rcp example from silabs gsdk 3.2.1

**Comment by cjszz2001** [comment]
What about your android chip-tool version?
Is the the OTBR running on RaspberryPI through wlan0 or eth0?
Thanks

**Review by jmeg-sfy — APPROVED** [review]

**Comment by mkardous-silabs** [comment]
@cjszz2001 
The android tool is built with your local version of the repo. As far as i know, their is no version persay.
My otbr uses wlan0, but both could work depending on your network setup.

**Review by saurabhst — APPROVED** [review]

**Comment by cjszz2001** [comment]
Thanks Mkardous. I followed your setup. But still can not  do the commissioning with the android tool. I could do it through  the python controller.
Could you specify more details of your setup?

**Pull request merged**